### ISSUE TITLE
TS9999 "}" or "{" expected   

### ISSUE THREAD

**Issue by avernikoz** [palantir/tslint#3887]
### Bug Report

- __TSLint version__: "^5.9.1"
- __TypeScript version__: "^2.8.0-dev.20180302"
- __Running TSLint via__:  "Webstorm"
#### TypeScript code being linted

```ts

const Span = styled.span``

const Label = styled.label``

const Checkbox = styled.input`
  display: none;

  & + ${Label} ${Span} {
    display: inline-block;

    width: 18px;
    height: 18px;

    cursor: pointer;
    vertical-align: middle;

    border: 1.5px solid #909294;
    border-radius: 3px;
    background-color: transparent;
  }

  & + ${Label}:hover ${Span} {
    border-color: #4ed8da;
  }

  &:checked + ${Label} ${Span} {
    border-color: #4ed8da;
    background-color: #4ed8da;
    background-image: url('https://image.flaticon.com/icons/png/128/447/447147.png');
    background-repeat: no-repeat;
    background-position: center;
    background-size: 14px;
  }
`
```

with `tslint.json` configuration:

```json
{
  "extends": [
    "tslint-config-standard",
    "tslint-react",
    "tslint-immutable",
    "tslint-config-prettier"
  ],
  "rulesDirectory": ["tslint-plugin-prettier"],
  "rules": {
    "no-var-keyword": true,
    "no-object-mutation": true,
    "no-delete": true,
    "no-loop-statement": true,
    "no-method-signature": true,
    "no-implicit-dependencies": false,
    "no-submodule-imports": false,
    "ter-arrow-parens": false,
    "jsx-boolean-value": false,
    "member-access": false,
    "jsx-no-multiline-js": false,
    "jsx-no-lambda": false,
    "semicolon": false,
    "ordered-imports": false,
    "only-arrow-functions": [true, "allow-declarations", "allow-named-functions"],
    "promise-function-async": true,
    "await-promise": true,
    "curly": true,
    "no-arg": true,
    "no-bitwise": true,
    "no-conditional-assignment": true,
    "no-construct": true,
    "no-duplicate-super": true,
    "no-duplicate-variable": true,
    "no-empty": true,
    "no-eval": true,
    "no-dynamic-delete": true,
    "no-for-in-array": true,
    "no-invalid-template-strings": true,
    "no-misused-new": true,
    "no-return-await": true,
    "no-sparse-arrays": true,
    "no-shadowed-variable": true,
    "no-this-assignment": [true, { "allow-destructuring": true }],
    "no-unused-variable": true,
    "no-use-before-declare": true,
    "triple-equals": true,
    "switch-default": true,
    "cyclomatic-complexity": true,
    "no-duplicate-imports": true,
    "arrow-return-shorthand": [true, "multiline"],
    "array-type": [true, "array"],
    "binary-expression-operand-order": true,
    "class-name": true,
    "interface-name": [true, "always-prefix"],
    "match-default-export-name": true,
    "newline-before-return": true,
    "no-boolean-literal-compare": true,
    "no-unnecessary-callback-wrapper": true,
    "no-unnecessary-initializer": true,
    "no-unnecessary-qualifier": true,
    "object-literal-shorthand": true,
    "one-variable-per-declaration": true,
    "prefer-template": true,
    "variable-name": [
      true,
      "ban-keywords",
      "check-format",
      "allow-trailing-underscore",
      "allow-pascal-case"
    ]
  }
}

```

#### Actual behavior
Hi, I am new to React-tslint, but I have a problem with tslint, that I can't resolve.
Tslint throws an error:
`TS9999:} expected`
or, in some places
`TS9999:{ expected`

[Link to image](https://imgur.com/a/usmhBDa)

Is it problem with tslint or with styled-components?
How to resolve it?
Thanks in advance.
#### Expected behavior
Without an error

**Comment by giladgray** [comment]
@avernikoz `TS####` errors come from the TypeScript compiler itself. this is not a TSLint issue.
